### PR TITLE
Added new plugin for random excuses.

### PR DESCRIPTION
**Pull request by rakeshseal0**
## Description
Returns a random excuse for you.

## How Has This Been Tested?
Created local devenv and tested manually.

## Additional context

Add any other context or screenshots about the pull request here.

## Checklist

- [x] I have read, understood and followed our [Contributing Guide](https://github.com/pyhoneybot/honeybot/blob/master/CONTRIBUTING.md)
- [x] My code follows the code style of this project
- [x] I have updated the documentation accordingly
- [x] I added my country flag to the `README.md`

## Code of Conduct

By submitting this issue, you agree to follow our [Code of Conduct](https://github.com/pyhoneybot/honeybot/blob/master/CODE_OF_CONDUCT.md)

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

**Comment by rakeshseal0** [comment]
Added new plugin which returns random excuses.

**Review by MrTanoshii — CHANGES_REQUESTED** [review]
Small typo in `info.py`

File: src/honeybot/plugins/downloaded/excuse/info.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+NAME = "excuse.py"
+ORIGINAL_AUTHORS = ["Rakesh Seal"]
+
+ABOUT = """
+Returns a random excuse for you.
+"""
+
+COMMANDS = """
+>>> .excuse
+returns a random image url.
```
[the remark this answers is not in the document]
Comment (rakeshseal0) [reply src/honeybot/plugins/downloaded/excuse/info.py:10]: yep. lol :3

**Review by MrTanoshii — APPROVED** [review]

**Comment by MrTanoshii** [comment]
Note: We broke the `black` GitHub action check.

Can you please run `black <filename>` on the files you've added in #151 and #155 (you can do both here)

https://github.com/psf/black#readme

What it does is ensure that we are using the same coding style.
I apologize, I was being lazy and was going to do it myself after merging your PRs but it's better for you to do it

**Comment by rakeshseal0** [comment]
Thank you @MrTanoshii  for being such a nice reviewer.

**Comment by MrTanoshii** [comment]
No worries, thank you for the contributions.

Have a great day

**Pull request merged**